1.9/overview/architecture/node-types.md

```diff
@@ -0,0 +1,79 @@
+---
+post_title: Node Types
+nav_title: Node Types
+menu_order: 1
+---
+
+A DC/OS node is a virtual or physical machine on which DC/OS components run. DC/OS nodes are networked together to form a DC/OS cluster.
+
+A DC/OS cluster is composed of three types of nodes: masters, private agents, and public agents.
+
+![DC/OS Node Types](/docs/1.9/overview/architecture/img/dcos-node-types.png)
+
+
+## Master Nodes
+
+A DC/OS master node is a node that works together with other master nodes to manage the rest of the cluster.
+
+Master nodes contain the bulk of the DC/OS components, including most notably a Mesos master process.
+
+### Protected Zone
+
+Depending on distribution, deployment method, and infrastructure configuration, master nodes may be publicly accessible or in a network zone that restricts access to improve security. Common strategies include restricting master node access to the IP range of your offices and requiring [VPN](https://en.wikipedia.org/wiki/Virtual_private_network) access.
+
+### High Availability
+
+Multiple master nodes work together to provide high availability and fault tolerance.
+
+A cluster with only one master node is usable for development, but is not highly available and may not be able to recover from failure.
+
+### Leader Election
+
+Mesos performs [leader election](https://en.wikipedia.org/wiki/Leader_election) and routes incoming traffic to the current leader to ensure consistency.
+
+Like Mesos, several DC/OS master node components (e.g. Marathon &amp; Zookeeper) also perform independent leader election. This means that the leaders for different components may be on different master nodes.
+
+### Quorum
+
+To maintain consistency, a quorum (half plus one) of master nodes must be connected at all times.
+
+For example, having three master nodes allows one to be down; having five master nodes allows two to be down, allowing for failure during a rolling update. Additional master nodes can be added for additional risk tolerance.
+
+DC/OS currently only allows specifying the number of master nodes at install time, due mostly to the complexity of changing the quorum and configuration of multiple components with leaders on different nodes.
```

Comment:
Suggest: `The number of master nodes can only be specified during installation. This is because of the complexity of changing the quorum and configuration of multiple components with leaders on different nodes. This may change in the future.`